src/Microsoft.Diagnostics.Monitoring/Contracts/IDiagnosticServices.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring
+{
+    /// <summary>
+    /// Set of services provided by the monitoring tool. These are consumed by
+    /// the REST Api.
+    /// </summary>
+    public interface IDiagnosticServices : IDisposable
```

Comment:
Do you anticipate having more than one implementation of this interface or wanting to put the impl in a different assembly from the interface? I'd suggest we get rid of it and assume that any method in DiagnosticServices that is marked public is the defacto interface.

Reply:
I was hoping to write some tests for this at some point. Keeping the interface will make that a lot simpler.

Reply:
works for me : )